### PR TITLE
Add log size param to cybert streamz workflow

### PR DESCRIPTION
**Pull request by brhodes10**
Adding log size param to cybert streamz workflow. User can input avg log size for benchmark estimates on throughput (mb/s) and average batch size (mb)

### CODE REVIEW

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

**Comment by brhodes10** [comment]
rerun tests

**Comment by efajardo-nv** [comment]
rerun tests

**Review by bsuryadevara — APPROVED** [review]

**Review by mike-wendt — APPROVED** [review]
Approved changes for our codeowners files; will rely on team to verify content

File: examples/streamz/README.md [hunk 1]
```diff
@@ -87,7 +87,21 @@ docker exec cybert-streamz bash -c 'source activate rapids && $KAFKA_HOME/bin/ka
...
+--group_id streamz \
+--input_topic input \
+--output_topic output \
+--model_file /path/to/model.pth \
+--label_file /path/to/label.json \
+--cuda_visible_devices 0,1,2 \
+--poll_interval 1s \
+--max_batch_size 1000 \
+--data /path/to/dataset \
+--benchmark 20
```
Comment (BartleyR) [inline examples/streamz/README.md:103]: Is the location of the benchmark output configurable (or documented somewhere)?

**Review by BartleyR — APPROVED** [review]

File: examples/streamz/README.md (hunk 1 above)
Comment (brhodes10) [reply examples/streamz/README.md:103]: It's not configurable right now-  currently benchmark output is printed to the screen and it is captured in the docker container logs.

Comment (BartleyR) [reply examples/streamz/README.md:103]: That's fine for now, as long as it's captured somewhere that persists (e.g., the logs).

**Comment by brhodes10** [comment]
rerun tests

**Comment by efajardo-nv** [comment]
rerun tests

**Pull request merged**